app/src/main/java/org/inspire/breath/data/Patient.java

```diff
@@ -29,8 +29,10 @@ public String getLastName() {
         return this.lastName;
     }
 
+    public int getAge() { return this.age; }
+
     public void setVolunteerId(int volunteerId) {
```

Comment:
Couple of these methods still refer to volunteer. Could you rename these too please :)?

Reply:
ah my bad, tried to do this quickly as I was heading out the door!